src/core/Akka/Actor/Scheduler/HashedWheelTimerScheduler.cs

```diff
@@ -52,6 +52,9 @@ public HashedWheelTimerScheduler(Config scheduler, ILoggingAdapter log) : base(s
             if (SchedulerConfig.IsNullOrEmpty())
                 throw ConfigurationException.NullOrEmptyConfig<HashedWheelTimerScheduler>();
 
+            if (!Util.MonotonicClock.IsHighResolution)
+                Log.Warning("HashedWheelTimerScheduler depends on the availability of high resolution performance counter which is not available in this system");
```

Comment:
When would the high resolution timer not be available?

Reply:
Is it not able to provide one on the build server?

Reply:
No idea, but its there as a preventative measures, just in case this happens in the future